File: src/main/java/net/theevilreaper/vulpes/api/repository/NotificationRepository.java
package net.theevilreaper.vulpes.api.repository;

import io.micronaut.data.annotation.Repository;
import io.micronaut.data.repository.PageableRepository;
import net.theevilreaper.vulpes.api.model.NotificationModel;

@Repository
public interface NotificationRepository extends PageableRepository<NotificationModel, String> {
}

File: src/main/kotlin/net/theevilreaper/vulpes/api/repository/NotificationRepository.kt
package net.theevilreaper.vulpes.api.repository

import net.theevilreaper.vulpes.api.model.NotificationModel
import org.springframework.data.mongodb.repository.MongoRepository


/**
 * @author theEvilReaper
 * @version 1.0.0
 * @since
 **/

interface NotificationRepository : MongoRepository<NotificationModel, String>